.buildkite/scripts/run_models.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
Same question as other `run_models.sh` changes.

Reply:
Refer to other replies